### ISSUE TITLE
@use-gesture

### ISSUE THREAD

**Issue by gloompi** [alephjs/aleph.js#527]
Hello, I'm trying to use aleph with `@react-three/fiber` and `@react-three/drei`, I was able to create a canvas and everything looked good until I imported `drei`, now it's giving me errors that types from `@use-gesture` are missing.

Here is image:
<img width="1022" alt="image" src="https://user-images.githubusercontent.com/20446664/215226162-1a4e3b08-87f0-43fc-9beb-f710cbabddc0.png">

Would it be possible to add those missing types in `@use-gesture`?

**Comment by ije** [comment]
related to https://github.com/ije/esm.sh/issues/483, i'm looking into it

**Comment by gloompi** [comment]
@ije do you guys need help? I see a lot of potential in this project and would love to help if needed.

**Comment by ije** [comment]
i just fixed the esm.sh issue, now you could use `@use-gesture` fine.

**Comment by ije** [comment]
> @ije do you guys need help? I see a lot of potential in this project and would love to help if needed.

yeah, build a framework is a big challenge, takes a lot of time! currently i am a bit busy on other things so don't have too much time on this project, but i am trying to keep working on it, mainly update the docs
